src/Datasource/EntityInterface.php

```diff
@@ -24,6 +24,11 @@
  * comply with.
  *
  * @property mixed $id Alias for commonly used primary key.
+ *
+ * @method array getInvalid()
+ * @method mixed|null getInvalidField(string $field)
+ * @method $this setInvalid(array $fields, bool $overwrite = false)
+ * @method $this setInvalidField(string $field, $value)
```

Comment:
I actually see them now in InvalidPropertyInterface.

Reply:
`class Entity implements EntityInterface, InvalidPropertyInterface`
So we dont need them after all. Sry.